server/protocols/webadmin-integration-test/webadmin-integration-test-common/src/main/java/org/apache/james/webadmin/integration/vault/LinshareBlobExportMechanismIntegrationTest.java

```diff
@@ -158,63 +157,9 @@ void exportShouldShareTheDocumentViaLinshareWhenImapDelete() throws Exception {
 
         exportVaultContent(webAdminApi, EXPORT_ALL_HOMER_MESSAGES_TO_USER_1);
 
-        assertThat(user1LinshareAPI.receivedShares())
+        assertThat(user1LinshareAPI.listAllDocuments())
             .hasSize(1)
-            .allSatisfy(receivedShare -> assertThat(receivedShare.getDocument().getName()).endsWith(".zip"))
-            .allSatisfy(receivedShare -> assertThat(receivedShare.getSender().getMail()).isEqualTo(USER_1.getUsername()));
-    }
-
-    @Test
-    void exportShouldSendAnEmailToShareeWhenJmapDelete() {
```

Comment:
Why removing tests in general where we check that the targeted user received a mail after an export with Linshare? That behavior still happens no?

Reply:
Now we upload directly to user space and no trigger sending an email to user anymore.

Reply:
But how can the user knows when the admin fulfilled his request and sent him the zip then? If he needs to check his linshare every once in a while it's not practical?

Reply:
We just upload directly to **UserSpace** by delegation account then  **OpenPaas** will do the send email.